collection_manager/main.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import logging
+
+from flask import Flask
+from flask_restplus import Api
+
+from collection_manager.services import CollectionProcessor, CollectionWatcher, MessagePublisher
+from collection_manager.services.history_manager import SolrIngestionHistoryBuilder, FileIngestionHistoryBuilder
+
+logging.basicConfig(level=logging.INFO)
+logging.getLogger("pika").setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+
+flask_app = Flask(__name__)
```

Comment:
The flask dependencies should be removed. They were still needed at some point to marshal the yaml file but i think you are not using it anymore (which is good !)

Reply:
Cool, I've removed flask